plugins/services/src/js/constants/VolumeTypes.js

```diff
@@ -1,6 +1,25 @@
 var VolumeTypes = {
-  PERSISTENT: "Persistent",
-  EXTERNAL: "External"
+  DSS: {
+    name: "DC/OS Storage Service",
+    type: "DC/OS",
+    description: "A locally persistent volume pre-created by the operator according to a storage profile.",
+    recommended: true
+  },
+  PERSISTENT: {
+    name: "Local Persistent Volume",
+    type: "Persistent",
+    description: "A locally persistent volume based upon the physical disks installed in the agent which has the capabilities of a single disk."
+  },
+  EXTERNAL: {
+    name: "External Persistent Volume",
+    type: "External",
+    description: "A REX-Ray backed volume that is mounted on the agent from network attached storage."
+  },
+  HOST: {
+    name: "Host Persistent Volume",
+    type: "Host",
+    description: "A Host volume ...... needs copy blablabla"
```

Comment:
"needs copy" - we cant merge it like this (obviously)

Reply:
Correct, I will add `hold merge` thanks.